hack/ci-e2e-kind-upgrade.sh

```diff
@@ -183,17 +205,17 @@ export GARDENER_PREVIOUS_VERSION="$(cat $GARDENER_RELEASE_DOWNLOAD_PATH/gardener
 kind_up
 
 # export all container logs and events after test execution
-trap '{
-  rm -rf "$GARDENER_RELEASE_DOWNLOAD_PATH/gardener-releases"
-  export_artifacts "$CLUSTER_NAME"
-  kind_down
-}' EXIT
+# trap '{
+#   rm -rf "$GARDENER_RELEASE_DOWNLOAD_PATH/gardener-releases"
+#   export_artifacts "$CLUSTER_NAME"
+#   kind_down
+# }' EXIT
```

Comment:
This is probably a debugging leftover. Please revert this change.

Reply:
Thanks @ScheererJ 
Fixed